client/src/pages/Login.js

```diff
@@ -1,7 +1,106 @@
-import { Box } from "@material-ui/core";
+import { Box, Grid, Typography, TextField, Button } from '@material-ui/core'
+import React, { useState, useEffect } from 'react'
+import { Link } from 'react-router-dom'
+import useStyles from '../styles'
+import { useForm } from 'react-hook-form'
+import green from '@material-ui/core/colors/green'
+import FormGroup from '@material-ui/core/FormGroup'
+import FormControlLabel from '@material-ui/core/FormControlLabel'
+import Checkbox from '@material-ui/core/Checkbox'
 
-function Login() {
-  return <Box>Login Page</Box>;
+
+const Login = () => {
+    const classes = useStyles()
+
+    const { register, handleSubmit, formState: { errors }, watch, setValue } = useForm({
+        defaultValues: { remember: false }
+    })
+
+    const onSubmit = (formData) => {
+        console.log(formData)
+    }
+
+    return (
+        <Grid container item direction='column' xs={12} spacing={6}>
+            <Grid item className={classes.inlineCenter} xs={12}>
+                <Typography className={classes.formHeaderTitle}>Member login</Typography>
+                <hr size="0.5" width="3%" color={green['A400']}></hr>
+                <Box className={classes.formHeaderBody}>
+                    <span>New here? </span>
+                    <Link className={classes.formHeaderBodyLink} to='signup'>Sign Up</Link>
+                </Box>
+            </Grid>
+            <Grid container item xs={12} alignItems="center" direction='column'>
+                <form onSubmit={handleSubmit(onSubmit)} className={classes.formWidth}>
+                    <Grid container item xs={12} alignItems="center" direction='column' spacing={3}>
+                        <Grid container item xs={12} md={4}>
+                            <TextField
+                                fullWidth
+                                autoComplete='Off'
+                                label="Email address"
+                                variant="outlined"
+                                name='email'
+                                {...register('email', {
+                                    required: 'email is required',
```

Comment:
```suggestion
                                    required: 'Email is required',
```
It would probably look better in the UI to capitalize this, ditto throughout

Reply:
yes, Capitalizing looks better. I will do it.